Automattic/woocommerce-payments
# Fixes related to test mode prompt style and copy

Fixes #6778

#### Changes proposed in this Pull Request







* Fixes to the styling and copy of the "Set Up Real Payments" option on the account page and modal.

### Overview Page 

**Before:** 

<img width="718" alt="Screenshot 2023-07-24 at 14 22 53" src="https://github.com/Automattic/woocommerce-payments/assets/11770181/e449a3ca-8b99-4277-a593-5e900f7457de">

**After:**

<img width="737" alt="Screenshot 2023-07-24 at 14 21 24" src="https://github.com/Automattic/woocommerce-payments/assets/11770181/f090f1fe-3fb1-48b8-9543-9a4ac8ba4814">

### Modal

**Before:**

<img width="634" alt="Screenshot 2023-07-24 at 14 22 57" src="https://github.com/Automattic/woocommerce-payments/assets/11770181/25207d70-782c-415f-940a-2bd0616a6025">

**After:**

<img width="648" alt="Screenshot 2023-07-24 at 14 21 38" src="https://github.com/Automattic/woocommerce-payments/assets/11770181/686f5d38-e64d-47e5-a015-cb1bad3d4cce">



#### Testing instructions





* Check out this branch on your client.
* Update the `wcpay_accounts` table in `localhost:8087` to set the `account_type` field to `wcpay_old` (or similar) for any active accounts. (To trick the server into thinking no account is connected)
* Refresh the cache on your local store. Ensure `Enable Progressive Onboarding` is enabled.
* Onboard a new account, select the builder path to create a test account.
* After onboarding is finished, you should be taken to the `Payments > Overview` page.
* Confirm that the style and copy of the content in the `Set up real payments` section matches the designs in #6830. 

-------------------


**Post merge**
